spectral/tests/database.py

```diff
@@ -1,9 +1,9 @@
 '''
-Runs unit tests of functions associated with the ECOSTRESS database.
+Runs unit tests of functions associated with spectral databases.
 
 To run the unit tests, type the following from the system command line:
 
-    # python -m spectral.tests.ecostress
```

Comment:
It was actually 'database' instead of 'ecostress' even before me changes. This was mistake.